test/serializers/serializer_for_test.rb

```diff
@@ -26,6 +26,31 @@ def test_overwritten_serializer_for_array
         end
       end
 
+      class SerializerNotFoundTest < ActiveSupport::TestCase
+        class Post < ActiveRecord::Base; end
+        class PostSerializer < ActiveModel::Serializer; end
+        class Comment < ActiveRecord::Base; end
+
+        def setup
+          @previous_serializer_not_found_policy = ActiveModelSerializers.config.on_serializer_not_found
+          ActiveModelSerializers.config.on_serializer_not_found = -> (klass) { fail NameError, "Serializer for [#{klass}] not found" }
+        end
+
+        def teardown
+          ActiveModelSerializers.config.on_serializer_not_found = @previous_serializer_not_found_policy
+        end
+
+        def test_serializer_not_found_triggers_configured_policy
```

Comment:
can you change these to the `test 'serializer not found triggers configured policy'` test format.

Thanks!